Formula/erlang_ls.rb

```diff
@@ -0,0 +1,44 @@
+class ErlangLs < Formula
+  VERSION = "0.21.2".freeze
+  desc "Erlang Language Server"
+  homepage "https://erlang-ls.github.io/"
+  url "https://github.com/erlang-ls/erlang_ls/archive/refs/tags/#{VERSION}.tar.gz"
+  sha256 "8a0466f525091a483401a958c6c1f85cd3203c6103b0fbabf784be449031b42c"
+  license "Apache-2.0"
+
+  depends_on "erlang@23" => :build
+
+  resource "rebar3" do
+    url "https://github.com/erlang/rebar3/releases/download/3.18.0/rebar3"
+    sha256 "f3e2641be923ce23076ce4843ee61c63fb392bc6c44dc9d129e4b31f7e136ff0"
+  end
```

Comment:
Why not `depends_on "rebar3"`?

Reply:
I tried that before, that formula never worked for me: 
```Building escript...
escript: exception error: undefined function rebar3:main/1
  in function  escript:run/2 (escript.erl, line 758)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1
  in call from init:do_boot/3
make: *** [all] Error 127
/usr/local/Homebrew/Library/Homebrew/shims/shared/git --version
/usr/local/Homebrew/Library/Homebrew/shims/shared/curl --version
/usr/local/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
```

Reply:
That's strange, since the `rebar3` formula is the same version. We should work out why that's happening first.